src/main/java/gregtech/loaders/recipe/handlers/PartsRecipeHandler.java

```diff
@@ -563,13 +618,15 @@ public static void processRound(OrePrefix roundPrefix, Material material, IngotP
                     "fIh", 'I', new UnificationEntry(ingot, material));
         }
 
-        LATHE_RECIPES.recipeBuilder().EUt(VA[ULV]).duration(100)
+        LATHE_RECIPES.recipeBuilder()
                 .input(nugget, material)
                 .output(round, material)
+                .duration(100)
+                .EUt(GTUtility.scaleVoltage(VA[ULV], workingTier))
                 .buildAndRegister();
     }
 
-    private static int getVoltageMultiplier(Material material) {
+    private static long getVoltageMultiplier(Material material) {
```

Comment:
We need to figure out if we are ditching these methods or not, and manually changing the tier for the materials it would apply to.